lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -1225,19 +1224,17 @@ matchWitness(TypeChecker &tc,
 
     // Open up the witness type.
     witnessType = witness->getInterfaceType();
-    // FIXME: witness as a base locator?
```

Comment:
Oh I see. I would make the change in an earlier patch so we can test it independently. I think @DougGregor should look at this since I don't understand why we need the locator here.